go.mod

```diff
@@ -4,16 +4,16 @@ go 1.19
 
 require (
 	github.com/DanielTitkov/go-adaptive-cards v0.2.2
-	github.com/alecthomas/kong v0.2.17
+	github.com/alecthomas/kong v0.2.22
 	github.com/ghodss/yaml v1.0.0
 	github.com/go-resty/resty/v2 v2.3.0
 	github.com/gogo/protobuf v1.3.2
 	github.com/google/go-querystring v1.1.0
 	github.com/google/uuid v1.3.0
 	github.com/gravitational/kingpin v2.1.11-0.20190130013101-742f2714c145+incompatible
-	github.com/gravitational/teleport/api v0.0.0-20230210215333-64e23540e972 // tag v12.0.2
+	github.com/gravitational/teleport/api v0.0.0-20230222184511-9527b6518c92 // tag v12.0.2
```

Comment:
This isn't likely to be tag v12.0.2 anymore

Reply:
Thank you for noticing.  I undid this change so that we can continue to reference a tagged commit.